src/functions-and-arrays.js

```diff
@@ -1,38 +1,96 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
-
+function maxOfTwoNumbers(a, b) {
+  if (a >= b) {
+    return a;
+  } else  {
+    return b;
+  }   
+}
```

Comment:
General tip: providing your if statement is a one liner, you can skip the curly brackets

` if (arr.length === 0) return null;`

Also, we can actually write it like this (below), this way the function will always return b unless the condition is met. It's ok to write it the way you did, it's just a nice thing to know.

```
function maxOfTwoNumbers(a, b) {
  if (a >= b) {
    return a;
  } else  {
    return b;
  }   
}
```
Edit: Oh, I've just noticed you're doing it already in the next one.

Reply:
Thanks for your feedback. It is definitely good to learn tips and different ways to write which will be helpful in the future.